contextual_orchestrator/server.py

```
@@ -5694,6 +5846,85 @@ def do_POST(self) -> None:  # noqa: N802
+                                        agent.id,
+                                        file_agent_affinity_key(agent),
+                                    )
+                                )
+                            except urllib.error.HTTPError as exc:
+                                last_error = exc
+                                if exc.code == 413:
+                                    continue
+                                continue
+                            except FileContractError:
+                                raise
+                            except Exception as exc:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)